src/ComponentConcerns/ValidatesInput.php

```diff
@@ -106,11 +106,11 @@ public function validate($rules = null, $messages = [], $attributes = [])
 
         $this->shortenModelAttributes($data, $rules, $validator);
 
-        $validator->validate();
+        $validatedData = $validator->validate();
```

Comment:
Why not just override the `$data` variable? It is not used afterwards.

Reply:
I find this more clear.
It is no longer the _data_, but the _validated data_. The variable name reflects what is in it.
Off course, this can be changed to `$data`